extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -274,6 +276,26 @@ export class AmpIframe extends AMP.BaseElement {
     }
     this.attemptChangeHeight(newHeight);
   }
+
+  /**
+   * Builds a key-value object representing data-params-* attributes on the
+   * element.
+   * @returns {Object}
+   * @private
+   */
+  getDataParams_() {
+    const attributes = this.element.attributes;
+    const params = {};
+    for (let i = 0; i < attributes.length; i++) {
+      const attr = attributes[i];
+      const matches = attr.nodeName.match(/^data-param-(.+)/);
+      if (matches) {
+        const param = matches[1];
+        params[param] = attr.nodeValue;
```

Comment:
In the ads case, where we have something similar we run `dashToCamelCase` over the param names. What do you think, does that make sense here?
https://github.com/ampproject/amphtml/blob/873385d7f745f0c119cda6c9b665f75b7e350873/src/string.js#L22

Both options suck: With it you cannot make dashes and without it, everything will be lower case.